src/DataTable/README.md

```diff
@@ -1039,6 +1059,166 @@ a responsive grid of cards.
 };
 ```
 
+### Customizing number of Cards shown per row
+Use `columnSizes` prop of `CardView` component to define how many `Cards` are shown per row at each breakpoint.
+
+`columnSizes` is an object containing the desired column size at each breakpoint. The example below shows 1 `Card` per row at `xs` breakpoint, 2 `Cards` at `sm` and `md`, and 4 `Cards` at `lg` and higher. You can read more about the API at https://react-bootstrap.netlify.app/layout/grid/.
+
+```jsx live
+() => {
+  const columnSizes = {xs: 12, sm: 6, lg: 3};
+
+  const ExampleCard = ({ className, original }) => {
+    const { name, color, famous_for: famousFor } = original;
+		
+    return (
+      <Card className={className}>
+        <Card.ImageCap src="https://source.unsplash.com/360x200/?nature,flower" srcAlt="Card image" />
+        <Card.Body>
+          <Card.Header title={name} />
+          <Card.Section>
+            <dl>
+              <dt>Color</dt>
+              <dd>{color}</dd>
+              <dt>Famous For</dt>
+              <dd>{famousFor}</dd>
+            </dl>
+          </Card.Section>
+          </Card.Body>
+      </Card>
+    );
+  };
+
+  return (
+    <DataTable
+      itemCount={5}
+      defaultColumnValues={{ Filter: TextFilter }}
+      isFilterable
+			isSortable
```

Comment:
nit: extra indents

Reply:
It always happed for some reason, there is definitely something wrong with my IDE... updated, thanks!